### PR TITLE
Add GetEncodingParameters/GetEncodingParams to CryptoObject and Plaintext

### PR DESCRIPTION
**Pull request by ArseniyKholod**
solves #18

### CODE REVIEW

**Comment by ArseniyKholod** [comment]
@sloede, I added `GetEncodingParams` and  `GetEncodingParameters`, it seems to work. Should I also add test for `CryptoObject`? (not sure it can be done easily)

File: test/test.jl [hunk 1]
```diff
@@ -8,4 +8,15 @@ using Test
...
 
     @test batch_size == 8192
+
+    x = [0.25, 0.5, 0.75, 1.0, 2.0, 3.0, 4.0, 5.0]
+    @show ptxt = MyOpenFHE.MakeCKKSPackedPlaintext(cc[], MyOpenFHE.CxxWrap.StdVector(x), 1, 0,
+		                                   MyOpenFHE.CxxWrap.StdLib.SharedPtr{MyOpenFHE.ILDCRTParams{MyOpenFHE.ubint{UInt64}}}(),
+						   0)
+    @show MyOpenFHE.SetLength(ptxt[], length(x))
+    @show ep = MyOpenFHE.GetEncodingParams(ptxt[])
+    @show batch_size= Int(MyOpenFHE.GetBatchSize(ep[]))
```
Comment (sloede) [inline test/test.jl:18]:
```suggestion
    @show batch_size = Int(MyOpenFHE.GetBatchSize(ep[]))
```

File: test/test.jl [hunk 2]
```diff
@@ -8,4 +8,15 @@ using Test
...
+
+    @test batch_size == 8192
+
```
Comment (sloede) [inline test/test.jl:21]:
```suggestion
```

**Review by sloede — CHANGES_REQUESTED** [review]
Looks almost ready to merge (just some formatting)

File: src/cryptoobject.cpp [hunk 3]
```diff
@@ -10,5 +10,6 @@ void wrap_CryptoObject(jlcxx::Module& mod) {
         wrapped.method("GetCryptoContextProxy", [](WrappedT& w) {
             return openfhe_julia::CryptoContextProxy(w.GetCryptoContext());
           });
+	wrapped.method("GetEncodingParameters", &WrappedT::GetEncodingParameters);
       });
```
[the remark this answers is not in the document]
Comment (sloede) [reply src/cryptoobject.cpp]: Ah, now it looks like I indented too much?!? First picture is my suggestion,
image

this one is how I see it in the PR diff:
image

Maybe it is already properly formatted but just not displayed correctly?

**Review by sloede — APPROVED** [review]
LGTM, thanks a lot!

**Pull request merged**